Changelog.md

```diff
@@ -12,6 +12,7 @@ Compiler Features:
 
 Bugfixes:
  * ABI Encoder: When encoding an empty string coming from storage do not add a superfluous empty slot for data.
+ * CommonSubexpressionEliminator: Process assembly items in chunks with maximum size of 2000. It helps to avoid extremely time-consuming searches during code optimization.
```

Comment:
Oh sorry, one more change: "Common Subexpression Eliminator"

Reply:
@wechman Can you apply this change so that we can merge it?

Reply:
done